packages/fury-adapter-oas3-parser/lib/parser/oas/parseComponentsObject.js

```diff
@@ -67,7 +67,7 @@ function parseComponentsObject(context, element) {
     [R.T, createInvalidMemberWarning(namespace, name)],
   ]);
 
-  return parseObject(context, name, parseMember, element);
+  return parseObject(context, name, parseMember)(element);
```

Comment:
Just to make it clearer for the reviewer, `parseObject` is a function that takes some arguments to construct an object parser, and it returns another function that is the final parser that accepts the single argument of element.

This was done to make it simpler to provide optional properties, this pattern happens elsewhere in Ramda and the code-base so it isn't specific to this case (see `R.pipe`, `R.compose` and `pipeParseResult`).